src/models/room.js

```diff
@@ -194,11 +194,13 @@ export function Room(roomId, client, myUserId, opts) {
         const serializedPendingEventList = client._sessionStore.store.getItem(pendingEventsKey(this.roomId));
         if (serializedPendingEventList) {
             JSON.parse(serializedPendingEventList)
-                .forEach(serializedEvent => {
+                .forEach(async serializedEvent => {
                     const event = new MatrixEvent(serializedEvent);
+                    if (event.getType() === "m.room.encrypted") {
+                        await event.attemptDecryption(this._client._crypto);
+                    }
                     event.setStatus(EventStatus.NOT_SENT);
```

Comment:
what if the pendingEvent hasn't yet been encrypted, e.g the app got restarted mid-encryption?

Reply:
If that is the case at the moment my code will have an unencrypted version persisted in the local storage which is far from ideal. Thank you for pointing that out

I'm going to filter out unencrypted messages from ever being persisted to local storage